Library/Formula/libgda.rb

```diff
@@ -22,6 +22,8 @@ class Libgda < Formula
   depends_on "libgcrypt"
   depends_on "sqlite"
   depends_on "openssl"
+  depends_on "mysql" => :optional
+  depends_on "mysql56" => :optional
```

Comment:
@mikemcquaid this is what I meant the other day. I didnt want to have to add those two versions in here, however if I only add mysql i would install the latest mysql if i only have mysql56 installed